app/models/room.rb

```diff
@@ -0,0 +1,10 @@
+class Room < ApplicationRecord
+  belongs_to :user
+
```

Comment:
- [ ] According to the ERD the model room has a relation of one-to-many with the reservation model, Please consider adding the association in the model as well as the migration file 👍🏼